flow-server/src/main/java/com/vaadin/flow/server/communication/StreamReceiverHandler.java

```diff
@@ -326,7 +326,7 @@ protected void doHandleXhrFilePost(VaadinSession session,
         sendUploadResponse(response, success);
     }
 
-    private boolean handleFileUploadValidationAndData(VaadinSession session,
+    protected boolean handleFileUploadValidationAndData(VaadinSession session,
```

Comment:
```suggestion
    /**
     * Validate that stream target is in a valid state for receiving data
     * and send stream to receiver. Handles cleanup and error in reading stream
     *
     * @param session
     *         The session containing the stream variable
     * @param inputStream
     *         the request content input stream
     * @param streamReceiver
     *         the receiver containing the destination stream variable
     * @param filename
     *         name of the streamed file
     * @param mimeType
     *         file mime type
     * @param contentLength
     *         The length of the request content
     * @param node
     *         The owner of the stream
     * @return true if upload successful, else false
     * @throws UploadException
     *         Thrown for illegal target node state
     */
    protected boolean handleFileUploadValidationAndData(VaadinSession session,
```

Protected methods need to have javadocs.